documentation/adr/003-target-TSE-SE-AWS-reference-architecture.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page.njk
+status: Proposed
+date: git Created
+tags:
+  - adr
+order: 3
+title: 003 - Target the TSE-SE Reference Architecture 
+---
+
+**Date: {{ page.date.toISOString().substring(0,10) }}**
+
+**Status: {{ status }}**
+
+## Context
+During a review of the architecture inherited from the outgoing supplier, we questioned choices regarding account structure, networking, IPAM and the tenant concept. These concerns were addressed in subsequent iterations of the target architecture and shared with AWS for review.
+
+In the course of the review, AWS proposed that we should use one of their reference architecures as a starting point, namely TSE-SE. 
+
+The [Trusted Secure Enclaves - Sensitive Edition (TSE-SE) Reference Architecture](https://github.com/aws-samples/landing-zone-accelerator-on-aws-for-tse-se/blob/main/architecture-doc/readme.md) is a comprehensive, multi-account AWS cloud architecture targeting sensitive level workloads. The architecture was designed in collaboration with national security; defence; national law enforcement; and federal, provincial, and municipal government customers to accelerate compliance with strict and unique security and compliance requirements. The TSE-SE Reference Architecture was designed to help customers address central identity and access management, governance, data security, comprehensive logging, and network design/segmentation in alignment with security frameworks such as NIST 800-53, ITSG-33, FEDRAMP Moderate, CCCS-Medium, IRAP, and other sensitive or medium level security profiles.
+
+We have reviewed the TSE-SE reference architecture and believe it aligns well with Core Cloud goals and objectives. There are some areas that we need to explore in more detail and we expect these areas of the design to be the subject of future ADRs. These include, but are not limited to, IPAM, Shared VPC vs Hub and Spoke, Traffic Inspection, and Ingress.
```

Comment:
Could we link out to the goals and objectives here - even if it only works for internal staff? It likely also pushes us to maybe open source them...

Reply:
I think this is important to set up some consequences, e.g. because it's native tooling we don't need to be burdened by custom 3rd party tooling.

Reply:
I think what you've identified here is that we should also publish our goals and objectives. We will address this in subsequent work

Reply:
I think linking back to objectives, goals and principles will be worth doing but require those things to be published too and therefore we should come back to this.

Regarding Nick's comment I added a consequence

- Robost and secure target architecture that can be delivered via AWS LZA ([ADR 001](../001-use-aws-landing-zone-accelerator/)) without requiring additional custom tooling.